src/app/domains/chats/chats.helpers.tsx

```diff
@@ -0,0 +1,20 @@
+import { useSupabase } from '@/components/SupabaseProvider'
+import { useQuery } from '@tanstack/react-query'
+
+export function useGetChat(chatId: string) {
+  const { supabase } = useSupabase()
+
+  const { data, isLoading } = useQuery(['chat'], async () => {
```

Comment:
Tip for all your future query hooks: Spread down all query params, you might need them in real usage. Like this:

```js
const { data, ...query } = useQuery(.....)
// ...
return {
  chat: data,
  ...query
}